olive/model/hf_utils.py

```diff
@@ -209,7 +209,7 @@ class HFConfig(ConfigBase):
     model_class: str = None
     components: List[HFComponent] = None
     dataset: Dict[str, Any] = None
-    model_loading_args: HFModelLoadingArgs = None
+    hf_from_pretrained_args: HFFromPretrainedArgs = None
```

Comment:
since, it is under `hf_config`, I think the `hf_` prefix is redundant?

Reply:
Makes sense to me. I'll update this.